Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/MediaPlayerElementSample_NestedPage5.xaml

```diff
@@ -0,0 +1,68 @@
+﻿<Page x:Class="Uno.Gallery.Views.NestedPages.MediaPlayerElementSample_NestedPage5"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid>
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition Height="*" />
+		</Grid.RowDefinitions>
+
+		<utu:NavigationBar Content="MediaPlayerElement sample"
+						   MainCommandMode="Action"
+						   Style="{StaticResource NavigationBarStyle}">
+			<utu:NavigationBar.MainCommand>
+				<AppBarButton Click="NavigateBack"
+							  Style="{StaticResource AppBarButtonStyle}">
+					<AppBarButton.Icon>
+						<BitmapIcon UriSource="ms-appx:///Assets/Close.png" />
+					</AppBarButton.Icon>
+				</AppBarButton>
+			</utu:NavigationBar.MainCommand>
+		</utu:NavigationBar>
+
+		<ScrollViewer Grid.Row="1"
+					  Padding="16">
+			<smtx:XamlDisplay smtx:XamlDisplayExtensions.Header="Using a Playlist"
```

Comment:
wrap the MediaPlayerElement with a border, and use IgnorePath to ignore the StackPanel+TextBlocks?
and add the `InitializePlaybackList()` part between <!-- xml comment --> inside that newly added Border?

Reply:
Changes done